app.py

```diff
@@ -1,11 +1,17 @@
 from selenium import webdriver
 from selenium.webdriver.common.by import By
-
-PATH = "/opt/chromedriver/chromedriver"
+import keys
+import tweepy
+from datetime import datetime
 
 driver = webdriver.Chrome()
 
 driver.get("https://www.corelogic.com.au/our-data/corelogic-indices")
-print(driver.title)
-print(driver.find_element(By.CSS_SELECTOR, "#daily-indices .graph-row:nth-child(4) .graph-column:nth-child(3)").text)
-driver.quit()
\ No newline at end of file
+
+daily_index = driver.find_element(By.CSS_SELECTOR, "#daily-indices .graph-row:nth-child(4) .graph-column:nth-child(3)").text
```

Comment:
If this CSS selector ever breaks. Maybe we can have multiple backup selectors. That way it's a bit more robust.